libraries/AP_Mount/AP_Mount_Servo.cpp

```diff
@@ -163,6 +105,15 @@ void AP_Mount_Servo::update_angle_outputs(const MountTarget& angle_rad)
     _angle_bf_output_rad.y = angle_rad.pitch;
     _angle_bf_output_rad.z = yaw_bf_rad;
 
+    // do no stabilization in retract or neutral:
```

Comment:
I think it could be better to instead pass a flag or frame argument into the update_angle_outputs() method